hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/LazyCreateThroughPrecompileSuite.java

```diff
@@ -154,7 +154,7 @@ public List<HapiSpec> getSpecsInSuite() {
                 autoCreationFailsWithMirrorAddress());
     }
 
-    HapiSpec resourceLimitExceededRevertsAllRecords() {
+    private HapiSpec resourceLimitExceededRevertsAllRecords() {
```

Comment:
We should not change these unless it corrects an otherwise failing test.

Reply:
Agree, reverted.